src/core_loggers/logger.ts

```diff
@@ -70,12 +58,13 @@ export abstract class Logger {
   //////////////////////////////////////////////////////////////////////////////
 
   /**
-   * @description
-   *     Write a log message. All extended classes must implement this method.
-   *     See Drash.CoreLoggers.ConsoleLogger/FileLogger for examples.
+   * Write a log message. All extended classes must implement this method.
+   * See Drash.CoreLoggers.ConsoleLogger/FileLogger for examples.
    *
-   * @param any logMethodLevelDefinition
-   * @param string message
+   * @param logMethodLevelDefinition {@link ./interfaces/log_level_structures.ts#LogLevelStructure | Interface of Log}
```

Comment:
should we change all of the `logMethodLevelDefinition` descriptions to use this description?

Reply:
I just followed the TSDocs convention. Though I have no way to tell if it's the best practice...

Reply:
Haha alright. It’s fine then. I’m resolving this